src/sourmash/command_sketch.py

```diff
@@ -283,3 +291,223 @@ def translate(args):
 
     _add_from_file_to_filenames(args)
     _execute_sketch(args, signatures_factory)
+
+
+def _compute_sigs(to_build, output, *, check_sequence=False):
+    "actually build the signatures in 'to_build' and output them to 'output'"
+    save_sigs = sourmash_args.SaveSignaturesToLocation(output)
+    save_sigs.open()
+
+    for (name, filename), param_objs in to_build.items():
+        assert param_objs
+
+        # now, set up to iterate over sequences.
+        with screed.open(filename) as screed_iter:
+            if not screed_iter:
+                notify(f"no sequences found in '{filename}'?!")
```

Comment:
can we optionally keep track of these somewhere? For large-scale collections where we might be downloading genomes on the fly, it would be handy to catch any cases where the download was empty for some reason.

Reply:
this is an excellent idea, but I think this is the wrong place to catch that, because of the other options (`--output-commands` and `--output-csv`) which don't even look at the genome/protein file content.

I think it might belong in the preparatory step(s) that look at the files in order to build the input csv to `sketch fromfile`? Will think on't. LMK if you have thoughts. Will not resolve yet.

Reply:
preparatory step(s) that look at the files would be great

Reply:
OK! In the meantime, I thought more about this and I think maybe we should be failing here. I created this issue - https://github.com/sourmash-bio/sourmash/issues/1887 - so I'll resolve this and we can discuss things over there ;).